src/main/java/ch/uzh/ifi/hase/soprafs22/service/GameLogicService.java

```diff
@@ -40,21 +51,16 @@ else if (cardRank.equals(Rank.FOUR)) {
             possibleMoves.add(4);
             possibleMoves.add(-4);
         }
-        // TODO: implement cooperation with team member
-        // and killing on the way
-        // probably just a special number for the move
-        // which then will be processed as for loop of 1 distance moves
+       
         else if (cardRank.equals(Rank.SEVEN)) {
+            int holesTraveled = game.getHolesTravelled();
             // special move num
+            // FIXME: Do we actually need this?
             possibleMoves.add(107);
```

Comment:
@shmnrr Do we still need this? I think you added this as special move num, but I handled it differently.

Reply:
Yeah it was just a placeholder

Reply:
Okay I'll remove it then